src/core/script/disclosure/disclosures-group.js

```diff
@@ -21,6 +21,7 @@ class DisclosuresGroup extends Instance {
     this.addAscent(DisclosureEmission.RETRIEVE, this.retrieve.bind(this));
     this.addAscent(DisclosureEmission.REMOVED, this.update.bind(this));
     this.descend(DisclosureEmission.GROUP);
+    this._isGrouped = this.isGrouped;
```

Comment:
this._isGrouped n'est pas mis à jour au update donc pas changé au mutate

Reply:
j'ai déplacé l'instanciation de `this._isGrouped` dans la méthode `update()` pour qu'il soit changé au `mutate`